docs/accordproject-faq.md

```diff
@@ -17,42 +17,41 @@ While the Accord Project technology is targeted at the development of smart lega
+Accord Project Smart legal contracts should not be confused with so-called blockchain “smart contracts”, which are scripts that necessarily operate on a blockchain. On the blockchain, a smart contract is often written in a specific language like 'solidity' that executes and operates on the blockchain most notably used by Ethereum. Ethereum being a major platform for generating smart contracts, other blockchains also create contracts compatible with Ethereum-based ones so that they can be portable on the Ethereum network as well. It lives in a closed world. An Accord Project Smart Contract contains text based template that integrates with a data model and the Ergo language. The three components are integrated into a whole. Using Ergo an Accord Project Smart contract can communicate with other systems, it can send and receive data, it can perform calculations and it can interact with a blockchain making it highly versatile.
 
 ### What benefits do Smart Legal Contracts provide?
 
-Contractual agreements sit at the heart of any organization, governing relationships with employees, shareholders, customers, suppliers, financiers, and more. Yet contracts today are not capable of being efficiently managed as the valuable assets they are. Currently contracts exist as static text documents stored in cloud storage services, dated contract management systems, or even email inboxes. Often these documents are Word files or PDFs that can only be interpreted by humans. A smart legal contract, by contrast, can be interpreted by machines. 
+Contractual agreements sit at the heart of any organization, governing relationships with employees, shareholders, customers, suppliers, financiers, and more. Yet contracts today are not capable of being efficiently managed as the valuable assets they are. Currently contracts exist as static text documents stored in cloud storage services, dated contract management systems, or even email inboxes. Often these documents are Word files or PDFs that can only be interpreted by humans. A smart legal contract, by contrast, can be interpreted by machines.
 
-Smart Legal Contracts can be easily searched, analyzed, queried, and understood. By associating a data model to a contract, it is possible to extract a host of valuable data about a contract or draft a series of contracts from existing data points (i.e., variables and their values).
+Smart Legal Contracts can be easily searched, analyzed, queried, and understood even by machines being static-typed. By associating a data model to a contract, it is possible to extract a host of valuable data about a contract or draft a series of contracts from existing data points (i.e., variables and their values).
```

Comment:
I am not sure about "even by machines being static-typed"    maybe something like "and used in comptutational processes"